### PR TITLE
protocol-v2-types: group related code

### PR DESCRIPTION
**Pull request by iurii-ssv**
Browsing through this code I've noticed it would be easier to see the whole picture if we move things around a bit, which is what this PR does.

### CODE REVIEW

**Review by nkryuchkov — APPROVED** [review]

**Review by y0sher — APPROVED** [review]
lgtm

**Comment by moshe-blox** [comment]
@iurii-ssv keep in mind that any change in `types`, `runner` or `qbft` packages might affect the spec-alignment script, since it compares funcs and types in those packages to their equivalent in the `ssv-spec` repo

so we should make sure that the funcs and methods here were not being included in the script

to do that, you can cd to `scripts/spec-alignment` and run `./differ.sh` and then search in the table for the funcs or methods (methods with struct name, for example the method should appear as "OperatorSigner.GetOperatorID" in the table)

File: scripts/spec-alignment/differ.config.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
...
...3","2f70337ba7566a69","dd607a44e1341e6b","5210501625ac3de5","f786bf475b5085aa","18a66ed6e613d9c1","e8943e7741f6843d","276a489bd5a00032","ba3bba59f10bf6b","3c50ce0c8089d871","89ee72f6c610ab84","c92b95a85da2cb11","927ea6aed3f98f20","9338904026a0ce37","9683cfa19dc544a3","4d3fa2b8dfcb5f5b", "f19e9a2b295bcfb3", "b10199b2de6f03b8", "1afc17e358f9ca79","4b58762c0b433442","d293ec1bc61bb707","3e88c3b49d093605","4890ff80c88cc41d","5227ff3a225dd20d","81a60407a3a0ba80","db2ad807eb66254a","d308bd7c553ccdcf","bdaf172971637cbe","6ade9202843071fe","2fe8e14083997744","19c9a5362d1e1d3a","5956f803d239f178","92c55a4548a8b760","9a95524213bccfff","2f51a7338b86c229","e96966a281d74505","3ee479b9cbbc3a1d","82b392ba39c6c594","b9d2404e5c570019","24f528d85fb021f2","fe9609a785305d81","b0934079dcd986cc","a9c520a19b26049","d19a9403fd732d94","74a928f5dcb2fdd9","cbbfdb5e68cdac80","10e39d2ceda91f34","f99a004cf6697875","8fa5e8ebf7d223ec","6c80c145ba705243","fbabbc90d0b4178a","ab0c7f24e551ca6","af38a11cb8682c75","b110cba51df9f8d2","c4ff2ed3d20dc419","9295a5bb10efcec7","ab56ea44a75f898a","ff51ef26ab53ba58","df3771e2589008f9","106e5689655bcfc6","f90e0fb6883bff93","667656095cec39ee","9a5597af260c748a","875393173d59b0f2"]
```
Comment (iurii-ssv) [inline scripts/spec-alignment/differ.config.yaml:1]:
```
diff --git a/protocol/v2/types/operator.go@OperatorSigner b/types/operator_signer.go@OperatorSigner
index 875393173d59b0f2..2222222
--- a/protocol/v2/types/operator.go@OperatorSigner
+++ b/types/operator_signer.go@OperatorSigner
@@ -1,4 +1,4 @@
-OperatorSigner interface {
-	SignSSVMessage(ssvMsg *SSVMessage) ([]byte, error)
-	GetOperatorID() OperatorID
+OperatorSigner struct {
+	SSVOperatorSK	*rsa.PrivateKey
+	OperatorID	OperatorID
 }
\ No newline at end of file

```

**Comment by iurii-ssv** [comment]
> to do that, you can cd to scripts/spec-alignment and run ./differ.sh and then search in the table for the funcs or methods (methods with struct name, for example the method should appear as "OperatorSigner.GetOperatorID" in the table)

Right, I should run a pipeline on CI to verify (and update if necessary) the differ alignment. I've done this for other PRs but not this one.

Here is the corresponding [branch](https://github.com/ssvlabs/ssv/commits/ci-pipelines/protocol-v2-types-group-related-code) with CI pipelines passing (after I've added corresponding hash to `ApprovedChanges`), the attached diff itself in the comment above.

**Pull request merged**